### ISSUE TITLE
Initial state of card is backwards

### ISSUE THREAD

**Issue by pjb3** [sbycrosz/react-native-credit-card-input#43]
I am a using CreditCardInput with autoFocus and when the screen initially renders, the back of the card image is shown. As soon as I enter something, the front of the card image is shown. It doesn't flip to the front image, it just appears. As I type, when I get to CVV, it does flip to the back, as it should. It is just that when the screen is initially rendered, the back card image is shown, but it should be the front card image.

Here is the relevant code:

```js
<CreditCardInput
  autoFocus
  inputStyle={styles.creditCardInput}
  onChange={this.onChange}
/>

const styles = StyleSheet.create({
  container: {
    flex: 1
  },
  creditCardInput: {
    color: textColor
  }
})
```

I am using react-native-credit-card-input 0.3.3 and react-native-flip-card 3.3.3

**Comment by eliprodigy** [comment]
+1

**Comment by alexsorokoletov** [comment]
+1, but this happens only if the screen is first in router. If you re-navigate to the same screen, all is good.

**Comment by sbycrosz** [comment]
Should be fixed in 4.1
